README.md

```diff
@@ -257,7 +257,3 @@ sudo maildev -s 9999
 All emails sent can be viewed from `http://localhost:1080`.
 
 *******
-
-### Other tasks
-
-* [Updating dependencies and `npm-shrinkwrap.json` files](http://fxa.readthedocs.io/en/latest/local-development/getting-started/#updating-npm-shrinkwrap).
```

Comment:
this should be this now: https://mozilla.github.io/application-services/docs/accounts/local-development.html#updating-npm-shrinkwrap